go.mod

```diff
@@ -1,48 +1,46 @@
 module github.com/anuvu/zot
 
-go 1.14
+go 1.16
 
 require (
 	github.com/99designs/gqlgen v0.12.2
-	github.com/Microsoft/hcsshim v0.8.16 // indirect
+	github.com/Microsoft/hcsshim v0.8.17 // indirect
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
-	github.com/apex/log v1.4.0
+	github.com/apex/log v1.9.0
 	github.com/aquasecurity/trivy v0.0.0-00010101000000-000000000000
```

Comment:
We should revisit the trivy fork and update

Reply:
its good to update that in another PR, lot of things changed.